1.0/start-vault.sh

```diff
@@ -4,8 +4,6 @@ GOOGLE_PROJ=$1
 INSTANCE_NAME=$2
 INSTANCE_IP=$3
 ENV=${4:-fiab}
-VAULT_TOKEN=${5:-$(cat ./.vault-token-fiabftw)}
-export VAULT_TOKEN=$VAULT_TOKEN
 
```

Comment:
```suggestion
chmod 755 $PWD/vault/consul.json
```
The main permissions issue I'd run into was the vault/consul.json not able to be read by the docker image on the remote GCE instance. Since Git doesn't track read permissions, this change should help us be on the safe side to ensure the file is world-readable on the remote host.